pkg/processor/runtime/nodejs/runtime.go

```diff
@@ -89,13 +94,32 @@ func (n *nodejs) getEnvFromConfiguration() []string {
 	}
 }
 
-func (n *nodejs) getHandlerPath() string {
-	handlerPath := os.Getenv("NUCLIO_JS_HANDLER")
-	if handlerPath != "" {
-		return handlerPath
+func (n *nodejs) getHandler() (string, string, error) {
+	parts := strings.Split(n.configuration.Spec.Handler, ":")
+
+	handlerFile := "handler.js"
+	handlerName := "handler"
```

Comment:
1. `handlerFile` -> `handlerFileName`
2. seems like handlerName is always overridden. best declare it either as a var or an empty string

Reply:
Done.